src/content/lodash/function/constant/vanilla.js

```diff
@@ -0,0 +1,3 @@
+const return42 = () => 42
```

Comment:
I would probably show an implementation of `constant` instead of inlining return42 like this, for example:

```
const contant = arg => arg
const return42 = constant(42) 
```